src/Parallel/GlobalCacheDeclare.hpp

```diff
@@ -2,9 +2,7 @@
 // See LICENSE.txt for details.
 
 /// \file
-/// Forward-declares CProxy_GlobalCache which MutableGlobalCache needs, but
-/// GlobalCache is defined after MutableGlobalCache. Also forward declares
-/// ResourceInfo which the GlobalCache has an entry method for.
+/// Forward-declares ResourceInfo which the GlobalCache has an entry method for.
```

Comment:
is the forward declaration of CProxy_GlobalCache still needed?

Reply:
Yes I got errors if I didn't have it and didn't bother trying to track things down.